lib/build.nix

```diff
@@ -169,26 +175,28 @@ rec {
   # Get a development shell with the extension in PYTHONPATH. Handy
   # for running tests.
   torchExtensionShells =
-    path:
+    { path, rev }:
     let
-      shellForBuildSet = path: buildSet: {
-        name = torchBuildVersion buildSet;
-        value =
-          with buildSet.pkgs;
-          mkShell {
-            buildInputs = [
-              (python3.withPackages (
-                ps: with ps; [
-                  buildSet.torch
-                  pytest
-                ]
-              ))
-            ];
-            shellHook = ''
-              export PYTHONPATH=${buildTorchExtension buildSet { inherit path; }}
-            '';
-          };
-      };
+      shellForBuildSet =
+        { path, rev }:
+        buildSet: {
+          name = torchBuildVersion buildSet;
+          value =
+            with buildSet.pkgs;
+            mkShell {
+              buildInputs = [
+                (python3.withPackages (
+                  ps: with ps; [
+                    buildSet.torch
+                    pytest
+                  ]
+                ))
+              ];
+              shellHook = ''
+                export PYTHONPATH=${buildTorchExtension buildSet { inherit path rev; }}
+              '';
+            };
+        };
       filteredBuildSets = applicableBuildSets (readBuildConfig path) buildSets;
     in
     builtins.listToAttrs (lib.map (shellForBuildSet path) filteredBuildSets);
```

Comment:
I think

```suggestion
    builtins.listToAttrs (lib.map (shellForBuildSet { inherit path rev; }) filteredBuildSets);
```

Reply:
thanks for the fix! updated in the latest commit